learn_evm/yellow-paper.md

```diff
@@ -0,0 +1,146 @@
+
+# Ethereum Yellow Paper
+
+So, you want to read the yellow paper. Before we dive in, keep in mind that the yellow paper is out of date and some in the community might refer to it as being depreciated. Check out the [BRANCHES.md](https://github.com/ethereum/yellowpaper/blob/master/BRANCHES.md) file of the [`yellowpaper` repository on github](https://github.com/ethereum/yellowpaper) to stay up-to-date on how closely this document tracks the latest version of the Ethereum protocol. At the time of writing, the yellow paper is up to date with the Berlin hardfork which occurred in April 2021.
+
+For a more up-to-date reference, check out the [Ethereum Specification](https://ethereum.github.io/execution-specs/autoapi/ethereum/) which features a detailed description of each opcode *for each hardfork* in addition to reference implementations written in python.
```

Comment:
We can also refer to https://github.com/crytic/building-secure-contracts/blob/master/learn_evm/eips_forks.md, to get visibility on the EIPs included for each fork